### ISSUE TITLE
Static files not load and get timeout

### ISSUE THREAD

**Issue by FelipeOF** [Simonwep/java-express#4]
I'm trying to access index.html through the static folder at http://localhost: 8080/index.html, but I did not succeed.

```
try {
    app.use(Middleware.statics(getPublicPath()));
} catch (IOException | URISyntaxException e) {
    e.printStackTrace();
}

```
My method getPublicPath:

```
private static String getPublicPath() throws URISyntaxException {
    return new File(WebServer.class.getProtectionDomain().getCodeSource().getLocation().toURI()).getPath() + "/public";
}
```

My config public folder:

```
.
├── Main.class
├── animation
│   ├── AnimationImagens$OnAnimationListener.class
│   └── AnimationImagens.class
├── controller
│   └── Controller.class
├── public
│   ├── css
│   │   └── index.css
│   ├── index.html
│   ├── js
│   │   ├── index.js
│   │   ├── lib
│   │   │   ├── axios.min.js
│   │   │   ├── jquery-3.3.1.min.js
│   │   │   ├── jquery.inputmask.bundle.min.js
│   │   │   ├── primus.io.js
│   │   │   ├── simplepeer.min.js
│   │   │   └── socket.io.js
│   │   ├── primus.io.js
│   │   └── webRtc.js
│   └── webRtc.html
└── web
    └── WebServer.class

```

I get timeout in all requests, but i when use:

```
app.get("/", (req, res) -> {
    try {
        res.send(Paths.get(getPublicPath() + "/index.html"));
    } catch (URISyntaxException e) {
        e.printStackTrace();
    }
});
```
I get success and load only html, not js, css

My HTML:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Captive portal</title>
</head>
<body>
    <div class="login-page">
        <div class="form">
            <div id="form">
                <input id="name" type="text" placeholder="nome" />
                <input id="phone" type="text" placeholder="telefone" />
                <button id="login">login</button>
            </div>
            <div id="finish" style="display: none;">
                Aguarde 3 segundos
            </div>
        </div>
    </div>

    <!-- <script src="/js/index.js"></script> -->
</body>
</html>
```

**Comment by simonwep** [comment]
I've tried to reproduce it (passing an absolute path to `Middleware.statics`) and it works fine. What does `getPublicPath()` return? Please note: relative paths don't work,

**Comment by hedwiggggg** [comment]
Nope, it's not invalid; This issue is real, i ran into it too..
But i digged deep and i found the issue; I guess you are not using Linux as an OS, aren't you?
Because you create the file-path like this: `Path reqFile = Paths.get(root + "\\" + path);`
It's a windows like path thing and it seems, that Linux can't handle this;
After changing this to `Path reqFile = Paths.get(root + "/" + path);` it run's fine;

Edit: I would recommend using something like `File.separator`

**Comment by simonwep** [comment]
Thanks @hedwiggggg for finding a solution :) Fixed here: 2f80e3743673a271063f143027e129cc3ddb948b
And yes, I'm using Windows :/

**Comment by Dmunch04** [comment]
I've run into a similar problem, and cannot fix it. It works fine when not having any `script` tags in the HTML file, but at soon as I add them back it just keeps loading.
